docs/built-in-bpa-rules.md

```diff
@@ -157,14 +157,14 @@ Upgrade your Kubernetes service cluster to a later Kubernetes version to protect
 **Recommendation**: To [upgrade Kubernetes service clusters](https://docs.microsoft.com/azure/aks/upgrade-cluster), in the [Microsoft.ContainerService/managedClusters resource properties](https://docs.microsoft.com/azure/templates/microsoft.containerservice/managedclusters?tabs=json#managedclusterproperties-object), update the *kubernetesVersion* property, setting its value to one of the following versions (making sure to specify the minor version number): 1.11.9+, 1.12.7+, 1.13.5+, or 1.14.0+.
 #### Severity: 1
 
-### TA-000026: Service Fabric clusters should only use Azure Active Directory for client authentication
-Service Fabric clusters should only use Azure Active Directory for client authentication. A Service Fabric cluster offers several entry points to its management functionality, including the web-based Service Fabric Explorer, Visual Studio and PowerShell. Access to the cluster must be controlled using AAD.
+### TA-000026: Service Fabric clusters should only use AAD for client authentication
+Service Fabric clusters should only use Azure Active Directory (AAD) for client authentication. A Service Fabric cluster offers several entry points to its management functionality, including the web-based Service Fabric Explorer, Visual Studio and PowerShell. Access to the cluster must be controlled using AAD.
 
 **Recommendation**: [Enable AAD client authentication on your Service Fabric clusters](https://docs.microsoft.com/azure/service-fabric/service-fabric-cluster-creation-setup-aad)
 #### Severity: 1
 
-### TA-000027: Transparent Data Encryption on SQL databases should be enabled
-Transparent data encryption should be enabled to protect data-at-rest and meet compliance requirements.
+### TA-000027: TDE on SQL databases should be enabled
```

Comment:
preferred previous

Reply:
I think all of these are very common abbreviations, look at the title of this article for example: https://learn.microsoft.com/en-us/sql/relational-databases/security/encryption/transparent-data-encryption?view=sql-server-ver16 And also considering where these fields were used I think the shorter the better